c/tskit/trees.h

```diff
@@ -1818,6 +1816,29 @@ bool tsk_tree_position_prev(tsk_tree_position_t *self);
 int tsk_tree_position_seek_forward(tsk_tree_position_t *self, tsk_id_t index);
 int tsk_tree_position_seek_backward(tsk_tree_position_t *self, tsk_id_t index);
 
+// These are in the header for testing purposes only
+struct stat_matrix_site_indicies {
```

Comment:
typo probably should be, ``stat_matrix_site_indices``. Elsewhere we use the ``indexes`` plural form, FYI but whatever you prefer.

Reply:
I guess strictly speaking if we put this in the public header we should also put in a ``tsk_`` prefix so that we're fulfilling the contracts we specify in the docs. (Names space collisions are unlikely I agree, but we do say that all public symbols start with a ``tsk_`` prefix)

Reply:
Also might as well follow the usual conventions and ``typedef`` it as ``tsk_stat_matrix_site_indexes`` (rather than as a struct)